tests/unit/test_views.py

```diff
@@ -333,11 +337,6 @@ def test_csrf_failure(client):
     assert "CSRF Failed" in response.rendered_content
 
 
-def assert_logged_in(client):
```

Comment:
This had been bugging me. Thank you for tidying it and doing something more django specific.